apollo-router/src/json_ext.rs

```diff
@@ -576,6 +713,7 @@ fn iterate_path_mut<'a, F>(
 
 /// A GraphQL path element that is composes of strings or numbers.
 /// e.g `/book/3/name`
+/// TODO: impl display for PathElement
```

Comment:
Still required?

Reply:
nice catch, not required anymore! fixed in d14d077d0